src/cascade/saver/db/save_model_results_to_db.py

```diff
@@ -103,7 +96,7 @@ def _normalize_draws_df(draws_df, execution_context):
     return draws.drop(to_drop, "columns")
 
 
-def _write_temp_draws_file_and_upload_model_results(draws_df, execution_context):
+def _write_temp_draws_file_and_upload_model_results(draws_df, execution_context, saver=None):
```

Comment:
That's a good question. It's only for testing. I did this because the testing code monkeypatches save_results, and monkeypatching our wrapper for the import fails when we're in testing mode. We look for calls to getattr, and a getattr has to happen in order to monkeypatch.

Another way around this is to rewrite the tests so they don't monkeypatch. I was apparently worrying about this last night, because that's the answer I thought of. I'll take another look.

Reply:
I'm not really objecting. I was mostly just curious. I don't fully understand your explanation but it sounds like we should fix that behavior so that patching is easy rather than solving this problem independently in many places.